src/main.qml

```diff
@@ -129,53 +156,268 @@ Application {
+                                horizontalCenter: maxDisplay.horizontalCenter
+                            }
+                            horizontalAlignment: Text.AlignHCenter
+                            verticalAlignment: Text.AlignVCenter
+                            font {
+                                styleName: "Bold"
+                                pixelSize: Dims.w(5)
+                            }
+                            //% "Max:"
+                            text: qsTrId("id-max")
+                        }
+
+                        Text {
+                            id: maxArrow
+
+                            color: "#88ffffff"
+                            anchors {
+                                verticalCenter: maxDisplay.verticalCenter
+                                verticalCenterOffset: -Dims.h(9.8)
+                                horizontalCenter: maxDisplay.horizontalCenter
+                            }
+                            horizontalAlignment: Text.AlignHCenter
+                            verticalAlignment: Text.AlignVCenter
+                            font {
+                                styleName: "Light"
+                                pixelSize: Dims.w(10)
+                            }
+                            text: "\u25b2"
+                        }
+                    }
                 }
 
-                Icon {
-                    name: IconTools.getIconName(owmId.value)
-                    anchors.centerIn: parent
-                    width: Dims.w(33)
+                Rectangle {
+                    id: iconCircle
+
+                    width: Dims.w(24)
                     height: width
+                    radius: width/2
+                    color: "#00ffffff"
+                    anchors {
+                        centerIn: parent
+                        horizontalCenterOffset: -Dims.w(30)
+                    }
+
+                    Icon {
+                        id: iconDisplay
+
+                        name: IconTools.getIconName(owmId.value)
+                        width: Dims.w(24)
+                        height: width
+                        opacity: 1
+                        anchors.centerIn: parent
+                    }
                 }
 
-                Label {
-                    //% "Max:"
-                    text: "<h6>" + qsTrId("id-max") + "</h6>\n" + convertTemp(maxTemp.value)
-                    horizontalAlignment: Text.AlignHCenter
-                    verticalAlignment: Text.AlignVCenter
-                    anchors.bottom: parent.bottom
-                    anchors.top: parent.top
-                    anchors.right: parent.right
-                    width: Dims.w(33)
-                    font.pixelSize: Dims.l(6)
+                Text {
+                    id: tempBig
+
+                    property string tempLength: convertTempRaw(Math.round((maxTemp.value + minTemp.value) / 2))
```

Comment:
I'm not sure what to think about this :/ I guess it's kind of a useful information but probably we should extend the BLE profile to send information required to show a real value here.

OR, we make it a bit more explicit that it's a mean temperature. (Even though it's not really the mean, it's the mean between the min and the max, not over a long series of measurements)

So... yeah, idk. Delegating to other maintainers to decide what is best here.

Reply:
In discussion with @MagneFire we agreed that having the real current temperature available is highly desirable. The mean is currently placeholder data for the time when a more recent currentTemp or any 3rd data point is there to present.
My intention was to limit the iterations of design improvements into a single step and already now have a large bigTemp for design reasons.
For the meanwhile i choose the mean becuse that is the only 3rd data point i could abbreviate from the min/max. Another option would be to make max the bigTemp for the time being.

Reply:
Ok